### ISSUE TITLE
Error after some updates

### ISSUE THREAD

**Issue by sdousley** [barryvdh/laravel-form-bridge#31]
I performed some updates to my code, and all works perfectly fine in my development environment, but when I pushed it out onto my production environment, I'm seeing the below error:

Unable to load the "Symfony\Component\Form\FormRenderer" runtime in "bootstrap_3_layout.html.twig". (View: /path/to/view.blade.php) (View: /path/to/view.blade.php)

Any ideas why this might be? PHP versions between live and dev are the same, all be it dev is run on centos, where live is on gentoo.

Looks like it's being thrown in vendor/twig/twig/lib/Twig/Environment.php:663.

**Comment by sailingdeveloper** [comment]
#metoo

**Comment by sdousley** [comment]
for now (since I don't need them) I've got round this issue by removing all my @form($form) calls. I'll keep hunting round and see what I can find to try and resolve this.

**Comment by sailingdeveloper** [comment]
@sdousley and replacing them with ... ?

**Comment by sdousley** [comment]
nothing at present. literally just commenting them out in the blade template

**Comment by sailingdeveloper** [comment]
Well, that would work of course, but I kind of need those forms 😅

**Comment by sdousley** [comment]
Yeah, mine fortunately aren't 100% necesary for my system, just ways to request stuff (it's a personal site anyway).

**Comment by sdousley** [comment]
hmmm, for some reason, on my local environment, where it works, it's wanting to load Symfony\Bridge\Twig\Form\TwigRenderer, but on live, it's the Symfony\Component\Form\FormRenderer

**Comment by barryvdh** [comment]
Does it work with above patch? Try `0.4.x@dev`

**Comment by sdousley** [comment]
Just tested - and now getting a different error:

Target [Twig_LoaderInterface] is not instantiable while building [Barryvdh\Form\FormRenderer, Twig_Environment]. (View: <my_views>/menu.blade.php) (View: <my_views>/menu.blade.php)

**Comment by barryvdh** [comment]
Are you using twig or without twig?

**Comment by sdousley** [comment]
I'm just doing it all in blade templates - I haven't (knowingly) used twig

**Comment by barryvdh** [comment]
Okay, please try https://github.com/barryvdh/laravel-form-bridge/commit/38c09be844e884090e69ac88b47039cb328900f4

**Comment by sdousley** [comment]
OK, whilst the previous version stopped working on my dev environment - making testing easier, this issue has now flagged up a new issue - "Template "bootstrap_3_layout.html.twig" is not defined. "

This happens on dev and live.

**Comment by sdousley** [comment]
OK, so I added SymfonyFormProvider to my list of providers, and added the resources/views/form directory, now I'm getting

Target [Symfony\Bridge\Twig\Form\TwigRendererEngineInterface] is not instantiable while building [Symfony\Bridge\Twig\Form\TwigRenderer].

**Comment by sdousley** [comment]
I've just tested this on completely fresh Laravel 5.5 and 5.4 (using v0.3.7) and both still get this error with absolutely no other modifications.  Is there any updates on this?

**Comment by sdousley** [comment]
OK, I've done a little more investigation on this, and have found that the following package changes allow this package to work:

  - Updating doctrine/inflector (v1.2.0 => v1.1.0): Downloading (100%)         
  - Updating erusev/parsedown (1.6.4 => 1.6.1): Downloading (100%)         
  - Updating league/flysystem (1.0.41 => 1.0.34): Downloading (100%)         
  - Updating monolog/monolog (1.23.0 => 1.22.0): Downloading (100%)         
  - Updating mtdowling/cron-expression (v1.2.1 => v1.2.0): Downloading (100%)         
  - Updating paragonie/random_compat (v2.0.11 => v2.0.4): Downloading (100%)         
  - Updating ramsey/uuid (3.7.1 => 3.5.2): Downloading (100%)         
  - Updating swiftmailer/swiftmailer (v6.0.2 => v5.4.5): Downloading (100%)         
  - Updating symfony/polyfill-mbstring (v1.6.0 => v1.3.0): Downloading (100%)         
  - Updating symfony/debug (v3.4.0 => v3.2.3): Downloading (100%)         
  - Updating symfony/console (v3.4.0 => v3.2.3): Downloading (100%)         
  - Updating symfony/css-selector (v3.4.0 => v3.1.10): Downloading (100%)         
  - Updating symfony/finder (v3.4.0 => v3.2.3): Downloading (100%)         
  - Updating symfony/http-foundation (v3.4.0 => v3.2.3): Downloading (100%)         
  - Updating symfony/event-dispatcher (v4.0.0 => v3.2.3): Downloading (100%)         
  - Updating symfony/http-kernel (v3.4.0 => v3.2.3): Downloading (100%)         
  - Updating symfony/options-resolver (v4.0.0 => v3.2.3): Downloading (100%)         
  - Updating symfony/polyfill-intl-icu (v1.6.0 => v1.3.0): Downloading (100%)         
  - Updating symfony/intl (v4.0.0 => v3.2.3): Downloading (100%)         
  - Updating symfony/process (v3.4.0 => v3.2.3): Downloading (100%)         
  - Updating symfony/polyfill-php70 (v1.6.0 => v1.3.0): Downloading (100%)         
  - Updating symfony/inflector (v4.0.0 => v3.2.3): Downloading (100%)         
  - Updating symfony/property-access (v4.0.0 => v3.2.3): Downloading (100%)         
  - Updating symfony/routing (v3.4.0 => v3.2.3): Downloading (100%)         
  - Updating symfony/translation (v3.4.0 => v3.2.3): Downloading (100%)         
  - Updating symfony/var-dumper (v3.4.0 => v3.2.3): Downloading (100%)         
  - Updating fzaninotto/faker (v1.7.1 => v1.6.0): Downloading (100%)         
  - Updating twig/twig (v2.4.4 => v2.1.0): Downloading (100%)         
  - Updating symfony/validator (v3.4.0 => v3.2.3): Downloading (100%)         
  - Updating symfony/twig-bridge (v3.4.0 => v3.2.3): Downloading (100%)         
  - Updating symfony/form (v3.4.0 => v3.2.3): Downloading (100%)         
  - Updating laravel/framework (v5.5.22 => v5.4.9): Downloading (100%)         
  - Removing barryvdh/laravel-form-bridge (dev-master 38c09be)
  - Installing barryvdh/laravel-form-bridge (dev-master 350b9f6): Downloading (100%)         
  - Installing symfony/dom-crawler (v3.1.10): Downloading (100%)

This requires changing to Laravel 5.4, as some of these packages don't work on 5.5 yet, so whilst this doesn't get it working on 5.5, it does mean that we can't use 5.5.

I think the most likely package that means that this can work would be the twig/twig or symfony/twig-bridge downgrade as on the later versions there's a few classes that phpStorm is flagging to me as deprecated

**Comment by edwinheij** [comment]
Did some digging but still got error ```"Template "bootstrap_3_layout.html.twig" is not defined. (View: ... )"``` thrown at ```vendor/twig/twig/lib/Twig/Loader/Chain.php``` line 86.

edit: I'm at Laravel 5.6.7

**Comment by ghost** [comment]
Any solution? (Laravel 5.6.15)
